script/package.ps1

```diff
@@ -2,8 +2,8 @@ param (
     [switch]$Dev = $false
 )
 
-$name = node -pe 'p = require(\"./package.json\"); `${p[\"name\"]}`'
-$version = node -pe 'p = require(\"./package.json\"); `${p[\"version\"]}`'
+$name = node -pe 'p = require(''./package.json''); `${p[''name'']}`'
+$version = node -pe 'p = require(''./package.json''); `${p[''version'']}`'
```

Comment:
What is the difference between `\"` and doing `''`? Does the former not work on your powershell?